home-application/src/main/java/com/softserveinc/ita/homeproject/application/service/impl/EntitySpecificationServiceImpl.java

```diff
@@ -0,0 +1,46 @@
+package com.softserveinc.ita.homeproject.application.service.impl;
+
+import java.util.Optional;
+import java.util.stream.Collectors;
+import javax.ws.rs.core.MultivaluedMap;
+
+import com.softserveinc.ita.homeproject.application.service.EntitySpecificationService;
+import io.github.perplexhub.rsql.RSQLJPASupport;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.stereotype.Service;
+
+@Service
+public class EntitySpecificationServiceImpl<T> implements EntitySpecificationService<T> {
+
+    public static final String DEFAULT_SORT = "id,desc";
+
+    public static final String RSQL_EQUAL = "==";
+
+    public static final String RSQL_EQUAL_OR = "=in=";
```

Comment:
Why do we need it? Seems like the change from not approved PR. Please avoid it. You should deliver only your changes as part of your PR.